changelogs/unreleased/gh-8391-bump-zstd.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Updated zstd to version pre-1.5.5 (gh-8391).
```

Comment:
I'm not sure it's safe to bump to an untagged version. I asked @kyukhin. Waiting for his response.